kroxylicious-bom/pom.xml

```diff
@@ -282,6 +287,11 @@
                     </licenseSets>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.github.git-commit-id</groupId>
+                <artifactId>git-commit-id-maven-plugin</artifactId>
+                <version>9.0.1</version>
+            </plugin>
```

Comment:
I think we should specify time zone UTC here (the tag to do so should be in the parent pom). Otherwise it's down to the TZ of the host running the release.